T4lysozyme-L99A_toluene/README.md

```diff
@@ -0,0 +1,3 @@
+# Restraints evaluation using  T4 lysozyme L99A-toluene complex
+
+Here you will find the yaml files used to compute the binding affinity of T4 lysozyme L99A-toluene complex using several types of restraints : harmonic, orientational and flat bottom restraints (with a well radius of 10 Å). 
```

Comment:
Do you also want to say that input files are available in the `input` subdirectory? And specify where these came from?

Sometimes this is dealt with by having a `## Manifest` subsection in your README.md that lists the different things available in the directory as bullet points `- text` in Markdown, with each bullet point having a brief description of the thing. e.g. see https://github.com/MobleyLab/drug-computing/blob/master/uci-pharmsci/README.md . Not required, just an idea in case it is helpful.